src/rapsV2/claimTransactionClaimableRap.ts

```diff
@@ -0,0 +1,13 @@
+import { createNewAction, createNewRap } from './common';
+import { RapAction, RapParameters } from './references';
+
+export async function createClaimTransactionClaimableRap(parameters: Extract<RapParameters, { type: 'claimTransactionClaimableRap' }>) {
+  let actions: RapAction<'claimTransactionClaimableAction'>[] = [];
+
+  const claim = createNewAction('claimTransactionClaimableAction', parameters.claimTransactionClaimableActionParameters);
+  actions = actions.concat(claim);
+
+  // create the overall rap
+  const newRap = createNewRap(actions);
```

Comment:
this could just be 

  const claim = createNewAction('claimTransactionClaimableAction', parameters.claimTransactionClaimableActionParameters);
  const newRap = createNewRap([claim]);



if I'm remembering correctly concat. actions was for raps with 2 or more actions

Reply:
yeah unnecessary for now but my next task after getting these merged is to add more actions to this rap so i think it's fine to keep it